### PR TITLE
update config.json file

### PR DESCRIPTION
**Pull request by RajARROW**
2 fixes

1) for API "nft/:address/:token_id/owners" change in the response body
 a) removed "status:SYNCING" from the response because API does not return this anymore
 b) changed response type to array from object as new API returns array not object

2) for API "nft/:address/owners" change in the response body
 a) removed "status:SYNCING" from the response because API does not return this anymore
 b) changed response type to array from object as new API returns array not object

### CODE REVIEW

**Comment by YosephKS** [comment]
> @YosephKS I think I have resolved the issue can you please have a look ?

Ah great thanks! I'll have a check @RajARROW

File: docs/configs/api-reference/configs.json [hunk 1]
```diff
@@ -2386,7 +2380,7 @@
               },
               {
                 "name": "result",
-                "type": "object",
+                "type": "array",
```
Comment (YosephKS) [inline docs/configs/api-reference/configs.json:2383]: Hey thanks for noticing this!

However to fix this it's a bit more complicated, which is why your build keeps failing 🙌🏼 

You'll need to wrap have an `object` type wrapped in the `array`, so will look something like this:
```js
{
    "name": "result",
    "type": "array",
    "field": {
        "type": "object",
        "fields" [
            // the rest of the fields
        ]
    }
```

File: docs/configs/api-reference/configs.json [hunk 2]
```diff
@@ -4512,7 +4500,7 @@
...
```
Comment (YosephKS) [inline docs/configs/api-reference/configs.json:4503]: Same issue, same fixed as explained above

File: docs/configs/api-reference/configs.json (hunk 1 above)
Comment (RajARROW) [reply docs/configs/api-reference/configs.json]: Thank you for pointing it out. Sorry for missing this. Fixed and pushed

**Pull request merged**